### ISSUE TITLE
Add pseudopotentials for the common workflows

### ISSUE THREAD

**Issue by giovannipizzi** [marvel-nccr/quantum-mobile#186]
At least for siesta, abinit, and Quantum ESPRESSO. Check [this comment](https://github.com/marvel-nccr/quantum-mobile/pull/185#issuecomment-829026550) and the other messages in the same thread, as well as the supplementary of the paper.

**Comment by ltalirz** [comment]
I think the pseudopotentials should be added in the relevant roles (and some roles already do this, e.g. siesta and QE).
Maybe add a checklist here of which ones are still missing?

**Comment by giovannipizzi** [comment]
Yes. I think the 'best' pseudos from the common-workflow protocols should be pre-installed.
@bosonie could you please prepare a simple bullet-point list of pseudos to install (and with which command) so @chrisjsewell can setup the various roles? I guess it's just a matter of summarising the SI of the paper here as bullet points (the info is probably also in the other issue linked above, but it has become very long so it's hard to find this information).

**Comment by bosonie** [comment]
Sorry for the late replay. The commands to run are:

- `aiida-pseudo install pseudo-dojo -v 1.0 -f jthxml`
- `aiida-pseudo install sssp -p efficiency`
- `aiida-pseudo install sssp -p precision`
- `verdi data psml uploadfamily /usr/local/share/siesta/psml-files-qm/nc-sr-04_pbe_standard/ nc-sr-04_pbe_standard_psml "pseudos from PseudoDojo"`

However this is for version 0.1.0 of aiida-common-workflows. The required pseudos (and the associated command) will surely change in the next release, therefore I would wait a bit before implementing this.
